nnbuilder-api/src/main/proto/tasksqueue.proto

```diff
@@ -0,0 +1,43 @@
+syntax = "proto3";
+
+package org.hse.nnbuilder.services;
+
+enum TaskName {
+  LearnNN = 0;
+  ApplyToData = 1;
```

Comment:
Make sure that your stored task object (curr. named TaskQueue) is generic enough to support this generalisation. How will you store args specific to each task type?